scrapers/ak/bills.py

```diff
@@ -479,40 +502,31 @@ def clean_action(self, action):
         action = action.replace("PREFILE RELEASED", "Prefile released")
 
         atype = []
-        if "READ THE FIRST TIME" in action:
-            atype.append("introduction")
-            atype.append("reading-1")
-            action = action.replace("READ THE FIRST TIME", "Read the first time")
-        if "READ THE SECOND TIME" in action:
-            atype.append("reading-2")
-            action = action.replace("READ THE SECOND TIME", "Read the second time")
-        if "READ THE THIRD TIME" in action:
-            atype.append("reading-3")
-            action = action.replace("READ THE THIRD TIME", "Read the third time")
-        if "TRANSMITTED TO GOVERNOR" in action:
-            atype.append("executive-receipt")
-            action = action.replace(
-                "TRANSMITTED TO GOVERNOR", "Transmitted to Governor"
-            )
-        if "SIGNED INTO LAW" in action:
-            atype.append("executive-signature")
-            action = action.replace("SIGNED INTO LAW", "Signed into law")
-        if "Do Pass" in action:
-            atype.append("committee-passage")
-        if "Do Not Pass" in action:
-            atype.append("committee-failure")
-        if action.startswith("PASSED"):
-            atype.append("passage")
-        if "(S) TRANSMITTED TO (H)" in action:
-            atype.append("passage")
-        if "(H) TRANSMITTED TO (S)" in action:
-            atype.append("passage")
-        if "REFERRED TO" in action:
-            atype.append("referral-committee")
-            action = action.replace("REFERRED TO", "Referred to")
-        if "Prefile released" in action:
-            atype.append("filing")
-        if "Approved by the Governor" in action:
-            atype.append("executive-signature")
+        for action_key in self._actions.keys():
+            # If we can detect a phrase that there is an OS action classification for
+            if action_key in action.lower():
+                # Our classification method for AK allows for more than one classification for certain actions
+                actions = self._actions[action_key]
+                [atype.append(this_action) for this_action in actions]
```

Comment:
`atype.extend(a for a in actions)`?

Reply:
oh I didn't know you could do it this way!